src/models/User.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const { DataTypes } = require('sequelize');
+const { sequelize } = require('../utils/db.js');
+
+const User = sequelize.define('user', {
+  name: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  email: {
+    type: DataTypes.STRING,
+    unique: true,
+    allowNull: false,
+  },
+  password: {
```

Comment:
Storing passwords in plain text can lead to major security vulnerabilities. It is essential to hash and salt the passwords for security reasons.